tests/lifecycle/helper/helper.go

```diff
@@ -186,3 +186,24 @@ func DeletePV(persistentVolume string, timeout time.Duration) error {
 
 	return nil
 }
+
+func AppendAnnotationsToPod(pod *corev1.Pod, annotation string, annotationValue string) {
+	m := make(map[string]string)
+
+	for k, v := range pod.Annotations {
+		m[k] = v
+	}
+
+	pod.Annotations = make(map[string]string)
+	pod.Annotations[annotation] = annotationValue
```

Comment:
This doesn't look like an "append" (add a value to the existing ones) operation to me, but a "set" one, overriding the existing annotations of the pod. Not sure whether that's exactly what you want/need here.
Other than that, you're creating and setting that "m" value but it's never used.
If you just want to append a new annotation to existing ones, you can do something like this:
```
if pod.Annotations == nil {
  pod.Annotations = make(map[string]string)
}
pod.Annotations[annotation] = annotationValue
```

Reply:
this is a draft that i forgot to delete, its not in use, deleted, thanks